### PR TITLE
Include more information about operations allowed on constant tables

### LINKED ISSUE #162: Add a quick sentence in the spec about reading entries from const tables
(tables with only static entries)

### PR DESCRIPTION
**Pull request by antoninbas**
Fixes #162

### CODE REVIEW

File: docs/v1/P4Runtime-Spec.mdk [hunk 1]
```diff
@@ -2404,6 +2404,22 @@ nonsensical values (such as an invalid action field). In the case of a *keyless*
 table (the table has an empty match key), the server must reject all attempts to
 `INSERT` a match entry and return an `INVALID_ARGUMENT` error.
 
+If the P4Runtime client attempts to perform any kind of write update on a
```
Comment (jafingerhut) [inline docs/v1/P4Runtime-Spec.mdk]: Minor nit: It is still legal to perform a write update on a table with `is_const_table = true` if it is a MODIFY of its default action, and the default action is not declared `const`.  Maybe some qualification of this statement could indicate that whether modifying the default action is legal is independent of the value of `is_const_table` ?

Comment (jafingerhut) [reply docs/v1/P4Runtime-Spec.mdk]: Or at least that is my understanding of current implementations in BMv2 and the way the specs are written, that this combination is possible.

Comment (antoninbas) [reply docs/v1/P4Runtime-Spec.mdk]: You're correct. The P4_16 spec actually includes an example table declaration with immutable entries but a non-const default action. I'll update my PR.

Comment (antoninbas) [reply docs/v1/P4Runtime-Spec.mdk]: I updated the paragraph and moved it to its own sub-section. PTAL.

**Comment by antoninbas** [comment]
> LGTM. There is one occurrence of P4_16 that I think Madoko markup format is something like P416

Fixed the P4_16 mention

**Pull request merged**